src/skin/launchimage.h

```diff
@@ -0,0 +1,22 @@
+#ifndef LAUNCHIMAGE_H_
+#define LAUNCHIMAGE_H_
+
+#include <QWidget>
+
+class QProgressBar;
+
+class LaunchImage: public QWidget {
+    Q_OBJECT
+  public:
+    LaunchImage(QWidget* pParent, const QString& styleSheet);
+    virtual ~LaunchImage();
+    void progress(int value);
```

Comment:
It just calls m_pProgressBar->setValue(value); which is probably thread save.
But that is not an issue here because the calling function is not thread save.
I will add a comment there.